.github/workflows/main.yml

```diff
@@ -81,7 +81,7 @@ jobs:
       - name: Setup gradle.properties
         run: |
           mkdir -p ~/.gradle
-          echo "org.gradle.jvmargs=-Xmx1024m -XX:MaxMetaspaceSize=768m -Dfile.encoding=UTF-8" >> ~/.gradle/gradle.properties
+          echo "org.gradle.jvmargs=-Xms2g -Xmx4g -XX:MaxMetaspaceSize=768m -Dfile.encoding=UTF-8" >> ~/.gradle/gradle.properties
```

Comment:
are we missing `org.gradle.daemon=false` ? because we seem to remove `--no-daemon` from some commands below.
or do we want to use the daemon for everything now?

Reply:
The latter, yes.